extensions/resteasy-reactive/rest-client-reactive-kotlin-serialization/deployment/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-rest-client-reactive-kotlin-serialization-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <artifactId>quarkus-rest-client-reactive-kotlin-serialization-deployment</artifactId>
+    <name>Quarkus - REST Client Reactive Kotlin Serialization - Deployment</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-reactive-kotlin-serialization-deployment</artifactId>
+        </dependency>
```

Comment:
This brings in the server parts, right? 

We really don't want to do that for the client as the client does not depend on the server